packages/nx-plugin/src/generators/executor/schema.json

```
@@ -46,6 +46,11 @@
       "type": "boolean",
       "default": false,
       "description": "Should the boilerplate for a custom hasher be generated?"
+    },
+    "skipLintChecks": {
+      "type": "boolean",
+      "default": false,
+      "description": "Do not eslint configuration for plugin json files."
```

this reads kind of funky I think you meant something alone these lines?
```suggestion
      "description": "Do not generate an eslint configuration for plugin json files."
```